ui-v2/src/components/dashboard/flow-runs-accordion/flow-runs-accordion-content.tsx

```diff
@@ -67,25 +68,46 @@ export function FlowRunsAccordionContent({
 	const flowRuns = data?.results ?? [];
 	const totalPages = data?.pages ?? 1;
 
+	// Prefetch a page of flow runs and their task run counts
+	const prefetchPage = useCallback(
+		(targetPage: number) => {
+			const filter = buildFilterForPage(targetPage);
+			const pageQuery = buildPaginateFlowRunsQuery(filter, 30_000);
+
+			void queryClient
+				.fetchQuery(pageQuery)
+				.then((data) => {
+					if (!data) return;
+					const flowRunIds = (data.results ?? [])
+						.map((run) => run.id)
+						.filter(Boolean);
+
+					if (flowRunIds.length === 0) return;
+
+					void queryClient.prefetchQuery(
+						buildGetFlowRunsTaskRunsCountQuery(flowRunIds),
+					);
```

Comment:
Unfortunately, this needs to be one prefect for each flow run ID to match the fetching behavior of the cards.